website/docs/tutorials/import-your-existing-apis/typegraph_std.py

```diff
@@ -0,0 +1,21 @@
+from typegraph import TypeGraph, policies
+from typegraph_std.github.ghes import import_ghes
+from typegraph_std.google.gmail import import_gmail
+
+with TypeGraph(
+    "emails",
+    # skip:next-line
+    cors=TypeGraph.Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
+) as g:
+    public = policies.public()
+
+    # {protocol}://{hostname}/api/v3
+    github = import_ghes({"protocol": "https", "hostname": "custom_host"})
```

Comment:
can we have a default setup for import_ghes? can we also use import_github so it is maybe clearer to someone not familiar with the package?